IsoCountries.swift

```diff
@@ -25,8 +25,8 @@ class IsoCountries {
     class func flag(countryCode: String) -> String {
         var string = ""
         var country = countryCode.uppercased()
-        for uS in country.unicodeScalars {
-            string.unicodeScalars.append(UnicodeScalar(127397 + uS.value)!)
+        for scalar in country.unicodeScalars {
+            string.unicodeScalars.append(UnicodeScalar(127397 + scalar.value)!)
```

Comment:
Yeah, I should unwrap this. 
About 127397 I actually found it on SO (https://stackoverflow.com/a/30403199 , see comments) is the base scalar where regional indicators starts, same as UnicodeScalar("🇦").value, minus 65 (UnicodeScalar("A").value)  or 

```swift
let regionalA = "🇦".unicodeScalars
let letterA = "A".unicodeScalars
let base = regionalA[regionalA.startIndex].value - letterA[letterA.startIndex].value
```

https://en.wikipedia.org/wiki/Regional_Indicator_Symbol
https://github.com/onmyway133/Smile/blob/master/Sources/Smile.swift#L52
http://www.codetable.net/decimal/127462
https://oleb.net/blog/2014/07/swift-strings/#regional-indicator-symbols